internal/configuration/configuration.go

```diff
@@ -22,31 +23,33 @@ var (
 
 func init() {
 	// service URLs for required services
-	settings["GetOpenShiftURL"] = Setting{"JC_OPENSHIFT_API_URL", "", []func(interface{}, string) error{util.IsURL}}
```

Comment:
!? We cannot do that. That defeats the purpose here. This are the actual names of the envirnoment variables.

Reply:
I am pretty sure you broke things with this change.